docs/releasenotes.md

```diff
@@ -0,0 +1,83 @@
+# Releasenotes Haal-Centraal-BRP-historie-bevragen
+
+
+## Versie 1.0.0
+
+Alhoewel er nog geen eerdere officiële versie van de Haal-Centraal-BRP-historie-bevragen API als release is uitgebracht zijn de specificaties van deze endpoints als onderdeel van de specificatie van Haal-Centraal-BRP-bevragen het afgelopen jaar beschikbaar geweest via de repository. In de [v0.9.0 release van BRP-bevragen](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v0.9.0) zijn de historie-endpoints nog opgenomen. In de afgelopen maanden hebben we de historie-endpoints in een aparte API ondergebracht. Daarbij zijn nieuwe inzichten die het afgelopen jaar zijn opgedaan opgenomen. Recentelijk hebben we de specificaties ook aangepast op het hergebruik van componenten uit BRP-Bevragen release 1.2.0.
+Dit is een globaal (en niet limitatief) overzicht van wijzigingen die daarvoor zijn doorgevoerd.
```

Comment:
"niet limitatief" betekent dit dat je mogelijk belangrijke wijzigingen niet hebt opgenomen? Met alleen "globaal" heb je jezelf namelijk al ingedekt tegen details.

Reply:
Als dat onduidelijkheid schept dan haal ik het weg. Zeg het maar.

Ik vermoed dat ik alle belangrijke wijzigingen wel heb benoemd, maar omdat ik niet zeker weet op welk punt eventuele partijen (die we niet kennen) ingestapt zijn bij het implementeren van deze functionaliteit weet ik ook niet zeker of ik alle voor hen relevante wijzigingen in beeld heb gebracht. Dat was al heel moeilijk voor de partij de we wel kennen.

Reply:
Ik zou "niet limitatief" weghalen, niet omdat het onduidelijk is, maar omdat je eigenlijk zegt dat de gebruiker deze inventarisatie alsnog zelf moet overdoen om eventuele andere wijzigingen ook mee te nemen.

Kunnen we niet aangeven vanaf wanneer we de wijzigingen hebben genoemd? (zeg 1 januari 2020)

Reply:
Ik haal "niet limitatief" weg. 

In de inleiding is al opgenomen dat ik de wijzigingen ten opzichte van de 0.9 versie van BRP-bevragen in beeld heb gebracht. Destijds waren de historie-endpoints nog onderdeel van BRP-bevragen. 
In de releasenotes staan ook nog enkele wijzigingen van voor die tijd (bv de component__embedded naar componentEmbedded), maar die laat ik er wel gewoon instaan.